source/ui.py

```diff
@@ -9,9 +9,58 @@
 See L{gui} for the graphical user interface.
 """
 
+import os
+import sys
+from ctypes import windll, byref, POINTER, addressof
+from comtypes import IUnknown
+from comtypes import automation 
+from logHandler import log
+import gui
 import speech
 import braille
 
+# From urlmon.h
+URL_MK_UNIFORM = 1
+
+# Dialog box properties
+DIALOG_OPTIONS = "dialogWidth:350px;dialogHeight:140px;resizable:yes;center:yes;help:no"
+
+#dwDialogFlags for ShowHTMLDialogEx from mshtmhst.h
+HTMLDLG_NOUI = 0x0010 
+HTMLDLG_MODAL = 0x0020 
+HTMLDLG_MODELESS = 0x0040 
+HTMLDLG_PRINT_TEMPLATE = 0x0080 
+HTMLDLG_VERIFY = 0x0100 
+
+def browseableMessage(message,title=None , isHtml=False):
+	"""Present a message to the user.
+	The message will be presented in an HTML document so that the user can read it with browse mode.
+	@param message: The message in either html or text.
+	@type message: str
+	@param title: The title for the message.
+	@type title: str
+	@param isHtml: Whether the message is html
+	@type isHtml: boolean
+	"""
+	htmlFileName  = os.path.realpath( 'message.html' )
+	if not os.path.isfile(htmlFileName ): 
+		raise LookupError(htmlFileName )
+	log.debugWarning("htmlMessage.html path: {}".format(  htmlFileName  ))
+	moniker = POINTER(IUnknown)()
+	windll.urlmon.CreateURLMonikerEx(0, unicode( htmlFileName ) , byref(moniker), URL_MK_UNIFORM)
+	if not title:
+		title = _("NVDA Message")
+	if not isHtml:
+		isHtmlArgument = "false"
+	else:
+		isHtmlArgument = "true"
+	dialogString = "{isHtml};{title};{message}".format( isHtml = isHtmlArgument , title=title , message=message ) 
+	log.debugWarning("dialog String = {}".format( dialogString ) )
+	dialogArguements = automation.VARIANT( dialogString )
+	gui.mainFrame.prePopup() 
+	result = windll.mshtml.ShowHTMLDialogEx( gui.mainFrame.Handle , moniker , HTMLDLG_MODELESS , addressof( dialogArguements ) , unicode(DIALOG_OPTIONS ), None)

```

Comment:
nit: Any reason you're assigning this to a `result` variable when it never gets used?